src/modules/esl-trigger/core/esl-trigger.ts

```diff
@@ -262,6 +276,7 @@ export class ESLTrigger extends ESLBaseElement {
     const target = this.$a11yTarget;
     if (!target) return;
 
+    typeof this.a11yLabel === 'string' && this.setAttribute('aria-label', this.a11yLabel);
```

Comment:
```suggestion
   target.setAttribute('aria-label', this.a11yLabel);
```
- ESL Trigger uses $a11yTarget term
- Not sure we need ignore value, propose remove the aria-label in case of empty config attribute passed